### PR TITLE
Update to Fast DDS [8577]

### PR DESCRIPTION
**Pull request by EduPonz**
Signed-off-by: EduPonz <eduardoponz@eprosima.com>

### CODE REVIEW

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/457/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/463/)

File: docs/fastrtpsgen/geninfo/geninfo.rst [hunk 1]
```diff
@@ -31,18 +31,18 @@ It also provides an initial implementation of a publisher and a subscriber using
...
-To compile *fastrtpsgen* java application, you will need to download its source code from
+To compile *Fast DDS-Gen* java application, you will need to download its source code from
 the `Fast-RPTS-Gen <https://github.com/eProsima/Fast-RTPS-Gen>`_ repository and with ``--recursive`` option and
 compile it calling ``gradle assemble``. For more details see :ref:`compile-fastrtpsgen`.
 
 .. code-block:: bash
 
     > git clone --recursive https://github.com/eProsima/Fast-RTPS-Gen.git
-    > cd Fast-RTPS-Gen
+    > cd Fast-DDS-Gen
```
Comment (IkerLuengo) [inline docs/fastrtpsgen/geninfo/geninfo.rst:45]: Should leave this as it was.
Cloning "Fast-RTPS-Gen" will still create a "Fast-RTPS-Gen" folder

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/464/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/465/)

**Pull request merged**